### PR TITLE
added KBaseAssembly read input types (and unit tests).

### PR DESCRIPTION
**Pull request by dcchivian**

### CODE REVIEW

**Comment by dcchivian** [comment]
9 of 9 unit tests are passing

File: lib/kb_maxbin/kb_maxbinImpl.py [hunk 1]
```diff
@@ -22,9 +22,9 @@ class kb_maxbin:
     # state. A method could easily clobber the state set by another while
     # the latter method is running.
     ######################################### noqa
-    VERSION = "1.0.0"
-    GIT_URL = "https://github.com/kbaseapps/kb_maxbin.git"
-    GIT_COMMIT_HASH = ""
+    VERSION = "1.0.2"
+    GIT_URL = "https://github.com/dcchivian/kb_maxbin"
```
Comment (sychan) [inline lib/kb_maxbin/kb_maxbinImpl.py]: Since we're going to merge it back in, we should revert it back to using the kbaseapps URL

Comment (dcchivian) [reply lib/kb_maxbin/kb_maxbinImpl.py]: sorry.  the make inserts that automatically

Comment (sychan) [reply lib/kb_maxbin/kb_maxbinImpl.py]: will it get fix automagically or do you need to update and push?

**Comment by dcchivian** [comment]
@sychan git url restored

File: lib/kb_maxbin/kb_maxbinImpl.py (hunk 1 above)
Comment (dcchivian) [reply lib/kb_maxbin/kb_maxbinImpl.py]: I just manually fixed it back now and pushed.  Should be part of PR now.

**Comment by Tianhao-Gu** [comment]
👍  Looking good. Tests passed.

**Pull request merged**